src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -214,6 +215,14 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
     };
 
   </script>
+
+  <div id="subscribeButton">
```

Comment:
You don't need this div, just put the id on the button itself. In fact, you don't need the id at all, you can just add an `onclick` handler when you're implementing the subscribe action.